doc/source/examples/unresolved-cfd-dem/single-particle-sedimentation/single-particle-sedimentation.rst

```diff
@@ -0,0 +1,414 @@
+        set rolling friction particles        = 0.1
+      end
+      set young modulus wall           = 1e6
+      set poisson ratio wall           = 0.25
+      set restitution coefficient wall = 0.97
+      set friction coefficient wall    = 0.3
+      set rolling friction wall        = 0.1
+    end
+
+Insertion Info
+~~~~~~~~~~~~~~~~~~~
+
+We use the list insertion method to insert the single particle in our domain. This is quite convenient, since we can tell exactly what is the position of the particle. We set the insertion frequency to 2000, which is the same as the time end. This way, the particle will be inserted at the top of the channel at the end of the DEM simulation.
+
+.. code-block:: text
+
+    subsection insertion info
+      set insertion method = list
+      set list x           = 0
+      set list y           = 0.08
+      set list z           = 0
+    end
+
+
+---------------------------
+Running the DEM Simulation
+---------------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``lethe-particles`` executable is within your path, the simulation can be launched on a single processor by typing:
+
+.. code-block:: text
+  :class: copy-button
+
+  lethe-particles initial-particle.prm
+
+.. image:: images/particle_insertion.png
+    :alt: inserted particle at the top of the channel
+    :align: center
+
+After the particle has been inserted it is now possible to simulate its sedimentation.
+
+
+-----------------------
+CFD-DEM Parameter File
+-----------------------
+
+The CFD simulation is to be carried out using the particle inserted within the previous step. We will discuss the different parameter file sections. We introduce the different sections of the parameter file ``single-particle-sedimentation.prm`` needed to run this simulation. Most subsections are explained in detail in other CFD-DEM examples such as:  `Gas-solid spouted bed <../../../examples/unresolved-cfd-dem/gas-solid-spouted-bed/gas-solid-spouted-bed.html>`_.
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation is run for :math:`2` s with a time step of :math:`0.005` s. The time scheme chosen for the simulation is first order backward difference method (BDF1). The simulation control section is shown:
+
+.. code-block:: text
+
+    subsection simulation control
+      set method               = bdf1
+      set number mesh adapt    = 0
+      set output name          = result_
+      set output frequency     = 20
+      set time end             = 2
+      set time step            = 0.005
+      set output path          = ./output/
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The physical properties subsection allows us to determine the density and viscosity of the fluid. We choose a density of :math:`996.8` kg/m\ :sup:`3` and a kinematic viscosity of :math:`0.0000008379` m\ :sup:`2`/s as to simulate the particle sedimentation in water.
```

Comment:
```suggestion
We set a density of :math:`996.8` kg/m\ :sup:`3` and a kinematic viscosity of :math:`0.0000008379` m\ :sup:`2`/s as to simulate the particle sedimentation in water.
```
Why take such specific values? I would have taken 1000 and 1e-6 for density and kinematic viscosity (which would be water at around 20 C). No need to change it, but it feels very specific. At least say you take the properties of water at 25C.